datasetinsights/io/config_handler.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import os
+import tempfile
+
+from yacs.config import CfgNode
+
+from datasetinsights.constants import GCS_BASE_STR
+from datasetinsights.io import create_downloader
+
+logger = logging.getLogger(__name__)
+
+
+class ConfigHandler:
+    """ Handles all the config related tasks such as
+     loading config from local or remote locations.
+    """
+
+    @staticmethod
+    def load_config(config):
+        """load config from local or remote location .
+
+        Args:
+            config: config location
+        Returns:
+            loaded config object of type dictionary
+        """
+        if config.startswith(GCS_BASE_STR):
```

Comment:
What if the config path starts with `http/https`? What if it starts with `/path/to/file`? What are different edge cases? What's the expected behavior if the file fail to download?

Reply:
In the following issue it was mentioned only GCS so I have implemented for GCS but now I have changed to handle HTTP/HTTPS also.
- https://github.com/Unity-Technologies/datasetinsights/issues/54


Supported locations:
1. GCS
2. HTTP/HTTPS
3. Local file: /a/cofig.yam

- If downloading or loading fails, run should terminate with caused exception.
- For HTTP/HTTPS, is it only public open URL or login URL also want to support? If yes then we need to add access token in train and evaluate cli.